scripts/patches/003-replace-curdir-with-dot.patch

```diff
@@ -0,0 +1,17 @@
+An instance of the `CURDIR` builtin variable was removed and simply replaced
+with a `.`. This replacement was performed since this builtin is not available
```

Comment:
`CURDIR` is available (it was `abspath` that wasn't available in the old version of make), but the problem is that if there's a space in the path, it doesn't work right. And using quotes, as in `SASS_LIBSASS_PATH = "$(CURDIR)"` didn't seem to work right in Windows.

Reply:
Ah yes... I'll make a correction to this comment. Thanks!